core/experiments/wp-cli.php

```diff
@@ -105,6 +105,27 @@ public function deactivate( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Experiment Status
+	 *
+	 * ## EXAMPLES
+	 *
+	 * 1. wp elementor experiments status container
+	 *      - This will return the status of Container experiment. (active/inactive)
+	 *
+	 * @param array $args
+	 */
+	public function status( $args ) {
```

Comment:
Please cover with tests

Reply:
The core team need to update the feature manager before. ( approved by Yotam )